zaza/model.py

```diff
@@ -39,7 +40,55 @@ def get_unit_from_name(unit_name, model):
     return unit
 
 
-async def run_in_model(model_name, f, add_model_arg=False, awaitable=True):
+def run(*steps):
+    """Run the given steps in an asyncio loop
+
+    :returns: The result of the asyncio.Task
+    :rtype: obj
```

Comment:
This should probably be:

```
:rtype: Any
```

To be forward compatible if/when sphinx does typing